src/components/utdragFraSykefravaeret/UtdragFraSykefravaeret.tsx

```diff
@@ -48,6 +48,21 @@ const StyledExpantionCardHeader = styled(ExpansionCard.Header)`
   }
 `;
 
+const KeyValueVisning = ({
+  keyword,
+  value,
+}: {
+  keyword: string;
+  value: string;
+}) => {
+  return (
+    <div className="text-base font-normal">
+      <b>{keyword}</b>
+      <span>{value}</span>
+    </div>
+  );
+};
```

Comment:
Er ikke så fan av så teknisk naming for noe som egentlig bare en en label og info som (forhåpentligvis) henger sammen med label 🤔
Kanskje `Info = ({ label, text })` er litt mer forklarende for en uvitende som plutselig finner en `KeyValueVisning`? Hva tenker du? 😊

Reply:
Ja, jeg fant aldri noe godt navn, så bra du kommer med forslag! Vi har lignende visning flere steder med <ord>: <annet ord>, der både key er bold og value vanlig, og value er bold og key vanlig 🥴 Så tanken var å egentlig lage noe generisk som man kunne bruke rundt omkring i appen. 
Info, label og text er nok bedre 👍🏼